pkg-new/preflights/host-preflight.yaml

```diff
@@ -1194,3 +1208,14 @@ spec:
           - fail:
               message: >-
                 The following directories lack execute permissions: {{ `{{ .Dirs | trim | splitList "\n" | join ", " }}` }}.
+    - textAnalyze:
+        checkName: Check filesystem on data directory path
+        fileName: host-collectors/run-host/xfs_info-data-dir.txt
+        regex: 'ftype=0'
+        outcomes:
+          - fail:
+              when: "true"
+              message: "The XFS filesystem on {{ .DataDir }} path is configured with ftype=0 which containers cannot run on. Please reconfigure the filesystem or use a different data directory."
+          - pass:
+              when: "false"
+              message: "File system on {{ .DataDir }} path is fit for use with containers"
```

Comment:
```suggestion
              message: "The XFS filesystem at {{ .DataDir }} is configured with ftype=1."
```

Reply:
Perhaps a nit here. A pass will also include all other file systems

Reply:
@banjoh are you saying that you would expand this same preflight in the future for additional hypothetical filesystem checks? If not, and we would add new checks in the future if we had to check something else for ext4 or zfs, then we'd probably have specific pass messages for those, so i think i'm in favor of what Alex is suggesting here. `"File system on {{ .DataDir }} path is fit for use with containers"` just seems like a bold claim to make when this check is very specific.